OneCommunityGlobal/HGNRest
# Kurtis fix new timer cleanup for dev (Do not review)

<h1 align="center"><img src="https://github.com/0xAbdulKhalid/0xAbdulKhalid/raw/main/assets/mdImages/about_me.gif" width ="25"> Kurt's Awesome timer fix </h1>

<h2>Description:</h2>
Azure was unable to be connected through pr430, so fixes were implemented to resolve this. Specifically, what was noticed was that "const port = 4500" was explicitly set. This was not the standard defining as noticed throughout the app. In addition to that, there were console.logs throughout the backend in order to track why the front end was unable to connect when pr430 was merged to the back end. These had to be cleared up as they didn't pass linting.

Fix port declaration to allow azure functionality of backend through process.env.PORT. Fix accompanying unused vars and console.logs by removing and throwing "throw new Error" instead


## Related PRS (if any):
To test this backend PR you need to checkout the <a href="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/979
">#979 frontend PR</a>.

…

## Main changes explained:
- change port declaration in server.js
- remove console.logs & replace with appropriate errors
- remove console.log from escapeRegexUnitTest
- remove unused vars
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any role of user
4. start timer
5. add tangible time (errors from .toLowerCase method being undefined should not occur)
6. following that, edit the time entry in "current week timelog" to confirm edit functionality
7. finally, delete it and confirm there are no errors as well
